src/api.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+const baseURL = 'https://api.github.com/';
+
+export const getRepos = (query) => {
+  return axios.get(`${baseURL}search/repositories`, {
+    params: {
+      q: query
+    }
+  })
+  .then(res => res.data.items
+    .map(({ id, name, description, html_url }) => ({id, name,description, html_url })
```

Comment:
yes, it was just a more general note

the method I was thinking is the one used in the example, `pick`

but this can stay as is for now